.github/workflows/pytest-dbutils-backup_db.yml

```diff
@@ -0,0 +1,65 @@
+---
+# This workflow is meant as a foundational workflow for running integration/unit tests on multiple targeted
+# ubuntu versions with multiple python versions.
+#
+# This workflow utilizes the build-dependency-cache workflow which sets up the environment dependencies using
+# bootstrap.py --all
+#
+
+# Documentation for the syntax of this file is located
+# https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions
+
+# The workflow name will show up in the action tab on github during execution
+# https://github.com/VOLTTRON/volttron/actions (or if you are pushing to your own fork change the user)
+name: Testing BackupDatabase
+
+# Determine what events are going to trigger a running of the workflow
+on: [pull_request]
+
+jobs:
+  build:
+    env:
+        TEST_FILE: backup_database.py
```

Comment:
I think this should be the whole filename....you will confuse people (me included) without the full file.

Reply:
Will do. I originally had the whole filename but then went with trying to abstract out the rest of the path. Will change to make it more explicit.